src/core/connect-extension/connect-extension-web.web.ts

```diff
@@ -0,0 +1,171 @@
+import { generateRandomEncryptionKey, decrypt } from '../secure/encrypt.web';
+import CryptoJS from 'crypto-js';
+import { v4 as uuidv4 } from 'uuid';
+import { IQRCodeConn, FirebaseRef, FIREBASE_BUCKET } from './types';
+import { storage, database } from 'firebase';
+import { updateReduxState } from '../../redux/wallets/actions';
+import { setExtensionStateLoaded } from '../../redux/ui/extension/actions';
+import { merge } from 'lodash';
+import { storeEncrypted, readEncrypted, deleteFromStorage } from '../../core/secure/storage.web';
+import { CONN_EXTENSION } from '../../core/constants/app';
+import Bowser from 'bowser';
+
+export const ConnectExtensionWeb = (() => {
+    let store: any = null;
+
+    const storeConnection = async (conn: IQRCodeConn) => {
+        try {
+            const encKey = generateRandomEncryptionKey().toString(CryptoJS.enc.Base64);
```

Comment:
since encKey is not used, jut send a dummy string there, there's no need to generate an encryption key that it's not used